__tests__/app.test.js

```diff
@@ -245,3 +245,107 @@ describe("POST: /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("PATCH: /api/reviews/:review_id", () => {
+  test("should return 200 and the updated review object when passed a valid positive increment", () => {
+    return request(app)
+      .patch("/api/reviews/2")
+      .send({ inc_votes: 4 })
+      .expect(200)
+      .then((review) => {
+        const reviewObj = review.body;
+        expect(reviewObj).toBeInstanceOf(Object);
+        expect(reviewObj).toHaveProperty("review");
+        expect(reviewObj.review.length).toBe(1);
+        const expectedOutput = {
+          review_id: 2,
+          title: expect.any(String),
+          review_body: expect.any(String),
+          designer: expect.any(String),
+          review_img_url: expect.any(String),
+          votes: expect.any(Number),
```

Comment:
have a look at what the test data votes look like and hardcode the value in to be more precise